pydoctor/test/test_epydoc2stan.py

```diff
@@ -1353,6 +1350,28 @@ def f(self, a: str, b: int):
 
     assert ''.join(docstring2html(B_f).splitlines()) == ''.join(docstring2html(A_f).splitlines())
 
+def test_cli_docformat_plaintext_overrides_module_docformat(capsys: CapSys) -> None:
+    """
+    Test if System.options.docformat overrides Module.docformat in the case of plaintext docformat only.
+    Issue https://github.com/twisted/pydoctor/issues/503.
```

Comment:
```suggestion
    When System.options.docformat is set to C{plaintext} it
    overwrites any specific Module.docformat defined for a module.
    
    See https://github.com/twisted/pydoctor/issues/503 for the reason
    of this behavior.
```

see https://jml.io/pages/test-docstrings.html